Dockerfile

```diff
@@ -1,12 +1,11 @@
-FROM ruby:2.5.0
+FROM ruby:2.5.1
 
 COPY update_gems.rb /usr/bin/update_gems.rb
 
 RUN wget -O- https://github.com/github/hub/releases/download/v2.2.3/hub-linux-amd64-2.2.3.tgz | tar zx
 RUN cp hub-linux-amd64-2.2.3/bin/hub /usr/bin/
 
-ENV GEM_HOME /usr/local/lib/ruby/gems/2.5.0
-RUN gem uninstall bundler
```

Comment:
gem uninstall bundler needs to stay in there I think

Reply:
we had an error because of that when some other PR removed that a while ago. Or is there a specific reason this should not be necessary anymore with 2.5.1?

Reply:
hummm I was getting "bundler is not installed in GEM_HOME" message when creating the image, that got fixed when I removed this line. https://hub.docker.com/r/schasse/gem_updater/builds/bc3avtemz5z8t8j6sbtz4ba/